exercises/robot-simulator/src/lib.rs

```diff
@@ -0,0 +1,32 @@
+#[derive(PartialEq, Debug)]
```

Comment:
The takeaway I get from this is: "If students think that they can't add more traits, we should add a comment telling them they can". I don't know whether the conditional is true (I can't read minds).